dummy/src/services/generateData.ts

```diff
@@ -0,0 +1,15 @@
+import fs from 'fs';
+import { AIResponse } from '../types/AIResponse';
+
+const exampleData: AIResponse[] = JSON.parse(
+    fs.readFileSync('example_data.json').toString()
+) as AIResponse[];
+
+const generateData = (): AIResponse => {
+    const time = new Date().valueOf();
+    const data = exampleData[Math.floor(Math.random() * exampleData.length)];
+    data.created = time;
+    return data;
```

Comment:
We might want to add the prompt string and current time in a human readable format to the returned data to make dubugging easier.

Reply:
Added new type: `DummyResponse`
This contains both the GPT3 format response, as well as additional data in the form of `DebugInfo`. It contains a timestamp as a UTC date, as well as the given prompt (including text and all other fields)